src/argilla/server/settings.py

```diff
@@ -73,6 +73,9 @@ class Settings(BaseSettings):
     elasticsearch_ca_path: Optional[str] = None
     cors_origins: List[str] = ["*"]
 
+    redis_host: str = "localhost"
+    redis_port: int = 6379
```

Comment:
Here we could use `pydantic.RedisDsn` type

Reply:
I have found the `RedisDsn` to be not really flexible if we want to specify attributes to the Redis URL so instead I'm using a `str` `redis` settings attribute where we can set the entire URL to connect to.